bundles/AdminBundle/Resources/public/js/pimcore/document/pages/settings.js

```diff
@@ -17,6 +17,10 @@ pimcore.document.pages.settings = Class.create(pimcore.document.settings_abstrac
     getLayout: function () {
 
         if (this.layout == null) {
+
+            /**
+             * @deprecated
```

Comment:
```suggestion
             * @deprecated 10.6 Will be removed in 11.
```
should we also add a `console.warn()` here?

Reply:
Yes, console.warn can also be added